code/repo_sync

```diff
@@ -294,10 +295,7 @@ def curl(url, destinationpath, onlyifnewer=False, etag=None, resume=False):
         print >> fileobj, 'fail'           # throw error if download fails
         print >> fileobj, 'dump-header -'  # dump headers to stdout
         print >> fileobj, 'speed-time = 30' # give up if too slow d/l
-        # next line does not work on windows
-        #print >> fileobj, 'output = "%s"' % tempdownloadpath
-
-        print >> fileobj, 'ciphers = HIGH,!ADH' # use only >=128 bit SSL
+        print >> fileobj, 'tlsv1'          # use only TLS 1.x
```

Comment:
Well, I the oldest one I have available is RHEL6 with NSS. I don't have the ability to test on RHEL5 or older.

Adding tlsv1 only disables sslv2 and sslv3 (but allows tlsv1, 1.1 and 1.2). Apple does not support sslv2 or sslv3 anyway, as per:
```
╭  :~/
╰ ➤ openssl s_client -connect swscan.apple.com:443 -ssl3
CONNECTED(00000003)
write:errno=54

╭  :~/
╰ ➤ openssl s_client -connect swscan.apple.com:443 -ssl2
CONNECTED(00000003)
write:errno=54
```